.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,32 @@
+name: D4L Utils CI (PR testing)
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - name: Display runner context
+      run: hostname -s
+    - uses: actions/checkout@v2
+    - name: Cache ruby gems
+      uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gem-
+    - run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+    - name: Reset iOS simulators (Required to avoid tests failing problem)
+      run: xcrun simctl erase all
```

Comment:
Not sure if we need this here... I think it was more a problem in the app because of the use of common and pods

Reply:
Sometimes those simulators break also in small unit tests, it doesnt add much to the timing so its safe to keep